apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-presenter.tsx

```diff
@@ -17,9 +25,13 @@ export const MultilingualTextPresenter = ({
         <>
             {/* TODO We need to separate the original in some way */}
             {items.map(({ languageCode, text, role }) => (
-                <div key={`${languageCode}-${role}`}>
-                    {`{${languageCode}} [${role}]`} {text}
-                </div>
+                <Box key={`${languageCode}-${role}`}>
+                    {text}
+                    <Typography
+                        variant="subtitle1"
+                        color={'secondary.dark'}
+                    >{`${languageCode}, '${role}'`}</Typography>
```

Comment:
This isn't really what I had in mind. How about this:
- pull the `defaultLanguageCode` from the config
- find the item with this language code (if it exists) in `items` and make this stand out on its own
- group all remaining items into an array called `translations`
- have a collapsible panel labelled `translations` that when you expand has one line of text for each translation and leverages your {`${languageCode}, '${role}'`} presenter (but don't use this presenter for the main text in the language with the `defaultLanguageCode`